webcompat/templates/contact.html

```diff
@@ -28,8 +28,8 @@ <h1 class="headline-1 heading">Say hello!</h1>
 			<h2 class="headline-2">Channels</h2>
 		    <ul>
 		    	<li>Join us
-		    		<a href="https://wiki.mozilla.org/IRC#Connect_to_the_Mozilla_IRC_server" title="Join us on IRC" target="_blank">
-		    			on IRC
+		    		<a href="https://wiki.mozilla.org/Matrix" title="Join us on Matrix" target="_blank">
+		    			on Matrix
```

Comment:
Maybe we can add a link to the room directly?

https://chat.mozilla.org/#/room/#webcompat:mozilla.org

Reply:
done